Marlin/Configuration.h

```diff
@@ -1364,6 +1364,10 @@
 //
 //#define REPRAP_DISCOUNT_SMART_CONTROLLER
 
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+
+//#define MAKEBOARD_MINI_2_LINE_DISPLAY_1602
+
 //
```

Comment:
Normally new panels are added at the end of the list so that people who are intimately familiar with the Config format can easily find the section start when scanning through it.  Regardless, the entry should also match the comment style:

//
// panel name
// link to external info on the panel (ideally reprap.org wiki page)
//
// any notes (optional)
//
// #define PANEL_DEFINE_NAME

Reply:
fixed